File: language-analysis/2.1.Concepts-Implementation/E-AbstractDataTypesAndEncapsulation/getter_and_setter_methods.py
class Course:
    def __init__(self, title):
        self._title = title

    def get_title(self):
        return self._title

    def set_title(self, title):
        self._title = title


course = Course("Programming Languages")
print(course.get_title())
course.set_title("Software Engineering")
print(course.get_title())

File: language-analysis/2.1.Concepts-Implementation/E-AbstractDataTypesAndEncapsulation/getter_and_setter_methods.js
class Course {
  constructor(title) {
    this._title = title;
  }

  getTitle() {
    return this._title;
  }

  setTitle(title) {
    this._title = title;
  }
}

const course = new Course("Programming Languages");
console.log(course.getTitle());
course.setTitle("Software Engineering");
console.log(course.getTitle());
